man/likelihood.Rd

```diff
@@ -88,6 +89,53 @@ likelihood(
   chains = chain_sizes, statistic = "size",
   offspring_dist = rpois, nsim_obs = 100, lambda = 0.5
 )
+# Example using an <epichains_tree> object
+set.seed(32)
+chains_tree_eg <- simulate_chains(
+ index_cases = 10,
+ pop = 100,
+ percent_immune = 0,
+ statistic = "size",
+ offspring_dist = rnbinom,
+ stat_max = 10,
+ generation_time = function(n) rep(3, n),
+ mu = 2,
+ size = 0.2
+)
+
+chains_tree_eg_lik <- likelihood(
+  chains = chains_tree_eg,
+  statistic = "size",
+  offspring_dist = rnbinom,
+  nsim_obs = 100,
+  mu = 2,
```

Comment:
```suggestion
  offspring_dist = rnbinom,
  mu = 2,
```

don't think we need `nsim_obs` here

Reply:
This has been fixed in 86afdf150ef537d275a2ca0439a43119df516359